[Docs-2.0] Update 10Commandments.md (#1050)
zkSNACKs/WasabiDoc: docs/using-wasabi/10Commandments.md

It also gives you time to move your coins to a different wallet if you notice that one of the two storage locations has been compromised.

> Wasabi is a fully non-custodial wallet, which means that you are always in possession of your keys, and this means you should safely store your backups.

Reviewer: ```suggestion
Wasabi is a fully non-custodial wallet, which means that you are always in possession of your keys, and thus you should safely store your backups.
```

If you have done this step, then even if your computer crashes, there is very little you have to worry about, you can always [recover your wallet](/using-wasabi/WalletRecovery.md).
Also, under no circumstance should you reveal the password or mnemonic words to anyone that asks for them.
Lastly, understand that if you lose your password, it becomes much harder (if not impossible) to restore your wallet, so store both safely!

## 4. Practice good labeling AND try to never reuse addresses

Each time you generate an address to [receive bitcoin](/using-wasabi/Receive.md) or you [send a transaction](/using-wasabi/Send.md), you must label the address with the observers and receivers of this transaction.
This label helps you use your bitcoin privately, and is stored exclusively on your device.

> Wasabi has strong automatic coin control features, and as you continue to use the wallet, it automatically coinjoins these coins to enable you to continue making payments without having to worry about revealing your transaction history.

Reviewer: ```suggestion
Wasabi has strong automatic coin control features, and as you use the wallet, it automatically coinjoins these coins to enable you to continue making payments without having to worry about revealing your transaction history.
```

Lastly, in the same way, that you should never receive bitcoin to the same address twice, you should try to avoid sending bitcoin to the same address twice.
So ask the recipients to provide you a new address for every transaction.

> ## 5. Be smart, be patient

Author: Sorry but I dont understand. Where is this refered?

> ## 5. Be smart, be patient

Reviewer: By editing that text the above link will not work anymore.

> ## 5. Be smart, be patient

Author: Ok but is there some other file refering to that link and wont there be a new one (https://docs.wasabiwallet.io/using-wasabi/10Commandments.html#_5-be-smart-be-patient) if this gets merged?

> ## 5. Be smart, be patient

Reviewer: Idk. It is not only about other files referring to that link, it is about external references to this link.

> ## 5. Be smart, be patient

Author: I think we cant avoid breaking some links, especially when the title doesn't make sense in WW2. The wallets decides when it's smart to coinjoin, unlike with WW1.
We should be careful with the links that external sites might be refering to but I dont think that's the case with this link.

> ## 5. Be smart, be patient

Reviewer: I think that title makes sense because the user has to decide when to run Wasabi if he/she wants to coinjoin, and plus the wallet doesn't decide when it is smart to coinjoin, it just coinjoins automatically when you have non-private coins.

> ## 5. Be smart, be patient

Author: I think it makes sense in the context outside Wasabi but not in it.
Depending on the profile, the wallet does decide when to coinjoin. Either with low fees, skipping rounds etc.
Not saying my suggestion is perfect but IMHO it's better than the old one, which just sounds deprecated.

> ## 5. Be smart, be patient

Reviewer: Saying `Be smart, be patient` is not really any better and can't be called a commandment.

> The process of engaging in a [coinjoin](/using-wasabi/CoinJoin.md) happens automatically by default to wallets that receive over 1M sat payment but it takes time.

Author: Changed it to + 500k. I think we should mention this because otherwise users would wonder why doesnt the wallet start mixing when they have for example 400k.

> The process of engaging in a [coinjoin](/using-wasabi/CoinJoin.md) happens automatically by default whenever you have more than 0.005 BTC in the wallet but it takes a while.

Reviewer: I don't get where did this 0.005 BTC come from.

I still think we shouldn't mention this detail, something like:
```suggestion
By default, the process of participating in a [coinjoin](/using-wasabi/CoinJoin.md) happens automatically in the background.
```

If necessary then I would change it to:

```suggestion
By default, the process of participating in a [coinjoin](/using-wasabi/CoinJoin.md) happens automatically in the background whenever you have more than 0.01 BTC non-private coins in the wallet.
```

Author: I'm afraid we will be asked to elaborate on these sections in the future but we'll see. Approved.

For coins to get coinjoined, you must keep your computer online and awake, because this is an interactive process.
As a coinjoin is just many users (up to 100) enqueuing coins, it may take up to one hour for you to successfully participate in a coinjoin.

> Once you've done so, you will see the amount of `Private coins` increase.

Reviewer: ```suggestion
Once you've done so, you will see the amount of `Private coins` increase in the main view.
```

As more users join the network, the frequency of these coinjoins will go up.

> Lastly, you can remix your coins without additional coordinator fee for up to 1 hop away.

Reviewer: ```suggestion
Lastly, you can remix your coins without additional coordinator fee for up to one hop away.
```

This provides you and the other coinjoin participants with plenty of additional privacy.

> ## 6. Never merge private and non-private coins

Author: Where is this refered or what do you mean?

> ## 6. Never merge private and non-private coins

Reviewer: https://docs.wasabiwallet.io/using-wasabi/10Commandments.html#_6-never-merge-mixed-and-unmixed-coins-and-avoid-large-merges-of-mixed-coins

This link will no longer work if we accept this change.

> ## 6. Never merge private and non-private coins

Author: I think we cant avoid breaking some links, especially when the title doesn't make sense in WW2. There's no mention of "mixed coins" in the wallet.
We should be careful with the links that external sites might be refering to but I dont think that's the case with this link.

> ## 6. Never merge private and non-private coins

Reviewer: Why do we need to change this? and why should we remove the `avoid large merges of mixed coins` part? It is still valid.

> ## 6. Never merge private and non-private coins

Author: > Why do we need to change this?

I was told we should avoid "mixing" if conviniently possible, to make it clear Wasabi coinjoin is different thing than centralized mixers.

> why should we remove the avoid large merges of mixed coins part?

Because merging coinjoined coins should not matter as much anymore, nor does the user know if he is consolidating when sending wtih private coins, right?

> ## 6. Never merge private and non-private coins

Author: @yahiheb thinking of it, I dont think we need to even mention anything about merging since it's complicated, as we established. So is this one good as is?

> Wasabi can't and shouldn't decide what users should do with their funds but it's highly recommended that users don't send payments using private and non-private coins in the same transaction.

Reviewer: There is no need for that first part.

```suggestion
It's highly recommended that users don't send payments using private and non-private coins in the same transaction.
```

> Wasabi can't and shouldn't decide what users should do with their funds but it's highly recommended that user don't send payments using private and labeled coins in the same transaction.

Reviewer: ```suggestion
Wasabi can't and shouldn't decide what users should do with their funds but it's highly recommended that users don't send payments using private and non-private coins in the same transaction.
```

> By merging your labeled coins with your coinjoined coins, you undo the privacy benefits of coinjoins, so be patient and if possible, wait till Wasabi automatically coinjoins more of your funds before making the payment.

Reviewer: ```suggestion
By merging your non-private coins with your private coins, you undo the privacy benefits of coinjoins, so be patient and if possible, wait till Wasabi automatically coinjoins more of your funds before making the payment.
```
